TemplateManagement/PowerShell/ActiveDirectoryJoinV2/Set-DomainJoinValuesToSecretStore-TemplateVM.ps1

```diff
@@ -134,5 +134,5 @@ Set-Secret -Name LabId -Secret $LabPrefix
 
 # Copy down files into the Public documents folder
 # TODO set the correct final location
-Invoke-WebRequest -Uri https://raw.githubusercontent.com/Azure/LabServices/domainjoinv2/TemplateManagement/PowerShell/ActiveDirectoryJoinV2/Join-Domain-StudentVM.ps1 -OutFile C:\Users\Public\Documents\Join-Domain-StudentVM.ps1
+Invoke-WebRequest -Uri https://raw.githubusercontent.com/Azure/LabServices/domainjoinv2/TemplateManagement/PowerShell/ActiveDirectoryJoinV2/Join-Domain-StudentVM.ps1 -OutFile "$env:PUBLIC\Documents\Join-Domain-StudentVM.ps1"
```

Comment:
Do we need 2 scripts for setting values to the secret store?  Perhaps we could only set the values that were passed in?  I can see customers getting confused about which of the two scripts they need?